### PR TITLE
READY: Use well-known cipher list

### LINKED ISSUE #104: Connect using authentication - Cipher list not correct #104  [JIRA: CLIENTS-640]
Hi, 

How can I connect to a cluster using authentication?

Currently I am using

     var nodes = [];
     hosts.forEach(function (host) {
            var hostPort = host.split(':');

            nodes.push(new Riak.Node({
                remoteAddress: hostPort[0],
                remotePort: hostPort[1],
                auth: {
                    user: "",
                    password: ""
                },
                connectionTimeout: 5000,
                cork: true
            }));

        });

        client = new Riak.Client(new Riak.Cluster({nodes: nodes}));

But I cant seem to get a connection so far ( RiakConnection Timed out trying to connect)

I can't seem to find documentation for connecting using credentials either. Is there documentation avialable? 

Looking forward to your reply

### PR DESCRIPTION
**Pull request by lukebakken**
Fixes #104 (CLIENTS-640) 

To test:

* Set up your Riak 2.1.3 `devrel` to use security:

```sh
git clone git://github.com/basho/riak-client-tools
cd riak-client-tools
devrel/setup-dev-cluster -s -p PATH/TO/riak/dev
```

* Clone `riak-nodejs-client` and, on `master` branch:

```
npm install
grunt security
```

* Check `error.log` from your first devrel node. You'll see the following:

```
no function clause matching tls_v1:enum_to_oid(28)
```

* Check out the `fixes/lrb/clients-640-cipher-list` branch

* Re-run `grunt security` and tests should pass

### CODE REVIEW

File: lib/core/riakconnection.js [hunk 1]
```diff
@@ -157,9 +160,9 @@ RiakConnection.prototype._receiveHealthCheck = function(data) {
 
 RiakConnection.prototype._receiveStartTls = function(data) {
     logger.debug("[RiakConnection]: receive StartTls response");
+    this._connection.removeListener('data', this._boundReceiveStartTls);
```
Comment (javajolt) [inline lib/core/riakconnection.js]: So you are now removing the listener no matter what?

**Comment by javajolt** [comment]
@lukebakken Had some problems setting up the cluster, so here's a tweak: https://github.com/basho/riak-client-tools/pull/2

File: lib/core/riakconnection.js (hunk 1 above)
Comment (lukebakken) [inline lib/core/riakconnection.js]: It makes more sense to do it this way but there is no need to change this for just a bug fix. I'll revert it.

**Comment by javajolt** [comment]
Works fine, now that I know the secret incantation

**Comment by javajolt** [comment]
:+1: 6aa7bf3

**Pull request merged**